parsers/postgresql/postgresql.go

```diff
@@ -209,7 +209,7 @@ func (p *Parser) handleEvent(rawEvent []string) *event.Event {
 
 	if rawDuration, ok := slowQueryMeta["duration"]; ok {
 		duration, _ := strconv.ParseFloat(rawDuration, 64)
-		ev.Data["duration"] = duration
+		ev.Data["duration_ms"] = duration
```

Comment:
I'm fully 👍🏻 with _adding_ a `duration_ms` field to honeytail events, but we shouldn't remove `duration` immediately.

Reply:
[afd2ba8](https://github.com/honeycombio/honeytail/pull/251/commits/afd2ba8fd5f8c4ad57c3c8bf2233de5dfae24a19)